Assets/AssetBookmarker/Editor/Project/Editor/GUI/ProjectBookmarkWindow.cs

```diff
@@ -89,20 +97,42 @@ private void OnGUI()
             {
                 this.RebuildBookmarkList();
             }
-            
+
             if (willRegisterAssets != null)
             {
-                var data = this.bookmarkDatas[this.currentBookmarkIndex]; 
+                var data = this.bookmarkDatas[this.currentBookmarkIndex];
                 data.Assets.AddRange(willRegisterAssets);
                 EditorUtility.SetDirty(data);
                 willRegisterAssets = null;
             }
 
+            prevIcon ??= EditorGUIUtility.IconContent("d_Profiler.PrevFrame");
```

Comment:
```suggestion
            prevIcon = EditorGUIUtility.IconContent("d_Profiler.PrevFrame");
```

Reply:
It's been a while, so I don't remember what version I checked it with, but it must have been a newer version with a higher C# language level. I have been using it like this in all project since I added that with no issues, but I can see it being an issue with older ones. No worries, though. I am perfectly fine with that modification.